### ISSUE TITLE
[Feature Request]: End of Run Quiz

### ISSUE THREAD

**Issue by Tipzilla** [besteon/Ironmon-Tracker#422]
### Tell us about your request

I thought it would be really cool if, similar to the statistics we see at the start of a run, at the end (game over or run completion) next to the "open log" button was a quiz button that players could participate in.

It could utilize tracked data from the run and ask questions like:

- What level was the first Blastoise you encountered on a trainer?
- Name two moves that enemy Porygon2 had at any point you encountered it?
- Did [your pokemon] try and learn [move]?
- What level was Brock's ace?
- Did you encounter a Kingdra at all? Yes/No

I'm not entirely sure if this is possible but I thought I would share the idea as I think it would be really need, you could even incorporate a points system. Because the run is over too, there's plenty of ways to utilize the tracker space to present this quiz.

Some questions could be cheesed by looking at the log, but for stream content I think this would be super fun to see!

**Comment by UTDZac** [comment]
This is a really sweet idea. I think a lot of people would have a ton of fun doing a little quiz after a good run.

Good starting questions too. We'll want to have a good selection of quiz questions to cycle through to keep things fresh. I'm all ears on this.

The one question about Brock's ace level won't work, since its always the same every seed (lv21).

**Comment by Tipzilla** [comment]
Yeah, once I thought about it I realized that question wouldn't work. An alternative could be "What Pokemon was Brock's ace?"

Although, it may be best (for simplicities sake) to just keep the questions to just yes and no.

Here's some more I've thought of for y/n:

- Did Misty have a Huntail?
- Did you encounter a Shedinja at any point?
- Was Ampharos's ability levitate?
- Did you heal at the Pokecenter x times?
- Was Blaine's TM Low Kick?

Again, the questions would be dependent on what information was tracked, if you fought a Ampharos and never used a ground move (thus never knowing it had levitate), you wouldn't get asked that question.
